package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-breakpoints-mixin",
+  "version": "0.0.1",
+  "description": "BreakpointsMixin for your React Components.",
+  "main": "dist/react-breakpoints-mixin.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/deseretdigital-ui/react-breakpoints-mixin.git"
+  },
+  "author": "Joshua Stoutenburg <jehoshua02@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/deseretdigital-ui/react-breakpoints-mixin/issues"
+  },
+  "homepage": "https://github.com/deseretdigital-ui/react-breakpoints-mixin",
+  "devDependencies": {
+    "autoprefixer-loader": "^1.0.0",
+    "css-loader": "^0.9.0",
+    "del": "^1.1.1",
+    "gulp": "^3.8.10",
+    "gulp-gh-pages": "^0.4.0",
+    "gulp-webpack": "^1.1.0",
+    "jsx-loader": "^0.12.2",
+    "react": "^0.12.1",
+    "require-dir": "^0.1.0",
+    "sass-loader": "^0.3.1",
+    "style-loader": "^0.8.2",
+    "webpack": "^1.4.13"
+  }
```

Comment:
Ah, nevermind, looks like you're using them in your example code and gulp tasks there. Sorry, was just looking at the main /src stuff. Forgot you do you /example fancy too.